docs/map-making/upgrading-1-8-maps.md

```diff
@@ -40,21 +40,23 @@ Within every game file (XML), inside the "games" folder inside the main folder o
 
 `="Naval Bombard Casualties Return Fire Restricted"`:`="Naval Bombard Casualties Return Fire"`
 
-### Special code XML changes for the above changes:
+`="SBR Affects Unit Production"`:`="Damage From Bombing Done To Units Instead Of Territories"`
 
-_(these are cases of advanced coding, so you probably don't have these)_
+### Changes of game (XML) file special code for the above items:
 
-Within every game file (XML), inside the "games" folder inside the main folder of the map, change all occurrences of the code before the colon with the code after the colon.
+_(these are cases of advanced coding, so you probably don't have these within the game file)_
 
-`value="conditionType" count="XOR"`:`value="conditionType" count="1"`
+Within every game file (XML), inside the "games" folder inside the main folder of the map, change:
 
-`value="conditionType" count="-reset-XOR"`:`value="conditionType" count="-reset-1"`
+`="conditionType" count="XOR"`:`="conditionType" count="1"`
 
-### Already deprecated but effective XML changes:
+`="conditionType" count="-reset-XOR"`:`="conditionType" count="-reset-1"`
 
-_(these items were already deprecated, so you shouldn't have had these in your games already)_
+### Changes of already deprecated game (XML) file items:
 
-Within every game file (XML), inside the "games" folder inside the main folder of the map, change all occurrences of the code before the colon with the code after the colon.
+_(these items were already deprecated, so you shouldn't have had these within the game file already)_
+
+Within every game (XML) file, inside the "games" folder inside the main folder of the map, change:
```

Comment:
IMO you can delete this and similar lines, the title already states that these changes are to the XML files.

Reply:
It might be overkill, but I'll leave them be since titles should not really have the job of explaining what to do, even though now I made them very descriptive (maybe it is the titles that are too descriptive and steal the job to the descriptions). Anyways, I think I'm not feeling to change either.